aws/resource_aws_vpn_connection_test.go

```diff
@@ -211,9 +212,63 @@ func TestAccAWSVpnConnection_tunnelOptions(t *testing.T) {
 				ExpectError: regexp.MustCompile(`can only contain alphanumeric, period and underscore characters`),
 			},
 
+			// Should pre-check:
+			// - local_ipv4_network_cidr
+			// - local_ipv6_network_cidr
+			// - remote_ipv4_network_cidr
+			// - remote_ipv6_network_cidr
+			// - tunnel_inside_ip_version
+			// - tunnel1_dpd_timeout_action
+			// - tunnel1_dpd_timeout_seconds
+			// - tunnel1_phase1_lifetime_seconds
+			// - tunnel1_phase2_lifetime_seconds
+			// - tunnel1_rekey_fuzz_percentage
+			// - tunnel1_rekey_margin_time_seconds
+			// - tunnel1_replay_window_size
+			// - tunnel1_startup_action
+			// - tunnel1_inside_cidr
+			// - tunnel1_inside_ipv6_cidr
+
 			//Try actual building
 			{
-				Config: testAccAwsVpnConnectionConfigTunnelOptions(rBgpAsn, "12345678", "169.254.8.0/30", "abcdefgh", "169.254.9.0/30"),
+				Config: testAccAwsVpnConnectionConfigTunnelOptions(
```

Comment:
Since this has so many arguments, it makes it difficult to keep track of which is which. If you create a new struct, you can pass all of these to the function as an object. Then it will be more clear what each value means at a glance.

Reply:
Done